models/DataObject/ClassDefinition/Data.php

```diff
@@ -654,6 +654,34 @@ protected function getPreGetValueHookCode(string $key): string
         return $code;
     }
 
+    /**
+     * @return string|null
+     */
+    public function getTypeHintInputType(): ?string {
+        return '?' . $this->getPhpdocInputType();
+    }
+
+    /**
+     * @return string|null
+     */
+    public function getTypeHintReturnType(): ?string {
+        return '?' . $this->getPhpdocReturnType();
+    }
+
+    /**
+     * @return string|null
+     */
+    public function getPhpDocInputType(): ?string {
+        return $this->getPhpdocType();
+    }
+
+    /**
+     * @return string|null
+     */
+    public function getPhpDocReturnType(): ?string {
```

Comment:
It's used as this (with lowercase 'd') everywhere else (including the interface).
```suggestion
    public function getPhpdocReturnType(): ?string {
```